ringvax/app.py

```diff
@@ -99,21 +143,68 @@ def app():
         "max_infections": max_infections,
     }
 
-    st.subheader(
-        f"R0 is {infectious_duration * infection_rate:.2f}",
-        help="R0 is the average duration of infection multiplied by the infectious rate.",
+    sims = []
+    with st.spinner("Running simulation..."):
+        tic = time.perf_counter()
+        for i in range(nsim):
+            sims.append(Simulation(params=params, seed=seed + i))
+            sims[-1].run()
+        toc = time.perf_counter()
+
+    st.write(
+        f"Ran {nsim} simulations in {format_duration(toc - tic)} with an $R_0$ of {infectious_duration * infection_rate:.2f} (the product of the average duration of infection and the infectious rate)."
```

Comment:
I think I'd actually like R0 to appear in the input sidebar

One day we might have a little toggle that allows you to either hold R0 fixed and set the infectious rate = R0/infectious period, or hold the rate constant and wiggle R0=rate*period (like we're doing now)

Reply:
It's a bit awkward in the bar, don't you think?

![image](https://github.com/user-attachments/assets/dce22227-4938-4533-ac08-84e16f974da4)

Reply:
Let's leave this for another PR

I have a vision that R0 could be displayed on a slider that is grayed/locked

Reply:
Breaking off as #37